Dependencies.cmake

```diff
@@ -4,6 +4,7 @@ if(BUILD_TESTING)
   FetchContent_Declare(
     googletest
     URL https://github.com/google/googletest/archive/ff233bdd4cac0a0bf6e5cd45bda3406814cb2796.zip
+    DOWNLOAD_EXTRACT_TIMESTAMP "ON"
```

Comment:
Good change I think, but changes to suppress CMake warnings (I assume(?) that's what this is) are not even mentioned in the PR description.

Reply:
I added a not to the PR description.